spec/index.bs

```diff
@@ -4039,9 +4039,14 @@ Issue(gpuweb/gpuweb#354): Finish defining multithreading API and add `[Serializa
 ### Shader Module Creation ### {#shader-module-creation}
 
 <script type=idl>
+dictionary GPUShaderModuleCompilationHints {
+    GPUPipelineLayout layout;
```

Comment:
should be required here

Reply:
```suggestion
    required GPUPipelineLayout layout;
```

(If we later wanted to make it optional when we added more fields, we could do so non-breakingly, but right now it doesn't make sense to not specify one.)